src/datasets/features/audio.py

```diff
@@ -342,5 +353,8 @@ def _decode_mp3_torchaudio(self, path_or_file):
     def _decode_mp3_librosa(self, path_or_file):
         import librosa
 
-        array, sampling_rate = librosa.load(path_or_file, mono=self.mono, sr=self.sampling_rate)
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", "PySoundFile.+?", UserWarning, module=librosa.__name__)
+            array, sampling_rate = librosa.load(path_or_file, mono=self.mono, sr=self.sampling_rate)
```

Comment:
maybe you can show the first warning this way:
```suggestion
            if _audioread_warned:
                warnings.filterwarnings("ignore", "PySoundFile.+?", UserWarning, module=librosa.__name__)
            else:
                _audioread_warned = True
            array, sampling_rate = librosa.load(path_or_file, mono=self.mono, sr=self.sampling_rate)
```